From 1ea23eef1437a152bfc54b56f65556b091644573 Mon Sep 17 00:00:00 2001
From: RickyWu-wiwynn <ricky_cx_wu@wiwynn.com>
Date: Wed, 17 Jul 2024 11:06:18 +0800
Subject: [PATCH 1/6] configuration: Revise CX7 NIC card temperature sensor

Description
1. Revise the CX7 NIC temperature sensor name to NICX_TEMP_C.
2. Revise the temperature threshold.
---
 configurations/cx7_ocp.json | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/configurations/cx7_ocp.json b/configurations/cx7_ocp.json
index 3f11519..9403a23 100644
--- a/configurations/cx7_ocp.json
+++ b/configurations/cx7_ocp.json
@@ -3,7 +3,7 @@
         {
             "Address": "$address",
             "Bus": "$bus",
-            "Name": "OCP CX7 NIC FRU $bus",
+            "Name": "OCP CX7 NIC FRU $bus % 24",
             "Type": "EEPROM"
         },
         {
@@ -12,15 +12,15 @@
             "Labels": [
                 "temp2"
             ],
-            "Name": "NIC Temp $bus",
-            "Name1": "NIC Temp $bus",
+            "Name": "NIC$bus % 24 REMOTE_TEMP_C",
+            "Name1": "NIC$bus % 24 TEMP_C",
             "Thresholds": [
                 {
-                    "Direction": "less than",
+                    "Direction": "greater than",
                     "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 115
                 },
                 {
                     "Direction": "greater than",
@@ -33,7 +33,7 @@
             "Type": "TMP421"
         }
     ],
-    "Name": "OCP CX7 NIC $bus",
+    "Name": "OCP CX7 NIC $bus % 24",
     "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Nvidia ConnectX-7 OCP3.0'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
-- 
2.44.2

